script/testing/util/test_server.py

```diff
@@ -85,28 +89,30 @@ def run_db(self):
         # Allow ourselves to try to restart the DBMS multiple times
         for attempt in range(constants.DB_START_ATTEMPTS):
             # Kill any other terrier processes that our listening on our target port
-            kill_processes_listening_on_db_port(self.db_port)
+            kill_pids_on_port(self.db_port)
 
-            self.db_output_fd, self.db_process = start_db(self.db_path,
```

Comment:
Is there a reason not to use `start_db`?

Reply:
Reverted back to `start_db`